### PR TITLE
Quote weather

### PR DESCRIPTION
**Pull request by manojncsoni007**
Description:
- Get client geolocation and show weather
- Show random quote

Screenshot:
image

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to be merged

File: src/Components/Quote/Quote.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import axios from "axios";
+
+const Quote = () => {
+    const [quoteText,setQuoteText] = useState('');
+
+    const getQuote = async () => {
+        try {
+            const response = await axios.get('https://type.fit/api/quotes');
+            setQuoteText(response.data[Math.floor(Math.random() * 1600)].text)
+            console.log(response);
```
Comment (arpit01923) [inline src/Components/Quote/Quote.jsx:12]: remove unnecessary console

**Review by arpit01923 — COMMENTED** [review]
- clean code
- good naming conventions
- good to merge

File: src/Components/Quote/Quote.jsx (hunk 1 above)
Comment (manojncsoni007) [reply src/Components/Quote/Quote.jsx:12]: Okay,will do

**Pull request merged**